components/integrations/commands_container/commands_container.tsx

```diff
@@ -22,51 +33,55 @@ const CommandRoute = ({component: Component, extraProps, ...rest}) => ( //eslint
     />
 );
 
-export default class CommandsContainer extends React.PureComponent {
-    static propTypes = {
-
-        /**
-        * The team data needed to pass into child components
-        */
-        team: PropTypes.object,
+type Props = {
 
-        /**
-        * The user data needed to pass into child components
-        */
-        user: PropTypes.object,
+    /**
+     * The team data needed to pass into child components
+     */
+    team?: Team;
 
-        /**
-        * The users collection
-        */
-        users: PropTypes.object,
+    /**
+     * The user data needed to pass into child components
+     */
+    user?: UserProfile;
 
-        /**
-        * Installed slash commands to display
-        */
-        commands: PropTypes.array,
+    /**
+     * The users collection
+     */
+    users?: RelationOneToOne<UserProfile, UserProfile>;
 
-        /**
-        * Object from react-router
-        */
-        match: PropTypes.shape({
-            url: PropTypes.string.isRequired,
-        }).isRequired,
+    /**
+     * Installed slash commands to display
+     */
+    commands: any[];
```

Comment:
There is a `Command` type in `mattermost-redux`

Reply:
done in my next commit